etc/config-template.toml

```diff
@@ -134,6 +134,30 @@
 ## When the pending write bytes exceeds this threshold, the "scheduler too busy" error is displayed.
 # scheduler-pending-write-threshold = "100MB"
 
+[storage.block-cache]
```

Comment:
have you missed other configurations for the shared cache in the config file?

Reply:
Discussed with @zhangjinpeng1987 and we think normal users may not need to change them and we should make them hidden. https://github.com/tikv/tikv/pull/4563#discussion_r278748010